scss/_membership.scss

```diff
@@ -20,35 +20,31 @@
         height: 572px;
         margin-right: 25px;
         text-align: center;
+        position: relative;
     }
     &__item:hover{
-        bottom: 20px;
-        left: 0;
-        right: 0;
         background-color: $content;
         color: $white;
         width: 400px;
         height: 572px;
         margin-right: 25px;
-        text-align: center;
+        bottom: 30px;
+        transition: all 0.5s;
```

Comment:
don't use transition all, is you want to animate background-color - write: transition: background-color 0.5s.